stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -1317,38 +1317,46 @@ function(add_swift_target_library_single target name)
   # Set compile and link flags for the non-static target.
   # Do these LAST.
   set(target_static)
-  if(SWIFTLIB_SINGLE_IS_STDLIB AND SWIFTLIB_SINGLE_STATIC AND NOT SWIFTLIB_SINGLE_INSTALL_WITH_SHARED)
+  if(SWIFTLIB_SINGLE_IS_STDLIB AND SWIFTLIB_SINGLE_STATIC)
```

Comment:
Wait, when is `SWIFTLIB_SINGLE_STATIC` set, but we're not building static libraries?

Reply:
Oh, when building `swiftCxx`. Neat.

Reply:
Yeah, I think it's the only "normal" case where that happens. The compatibility libraries use these flags also, I think, but they probably aren't being used as a dependency (`swiftCxxStdlib` presumably wasn't either until now).